sdk/storage/storage-queue/review/storage-queue.api.md

```diff
@@ -146,14 +146,14 @@ export function generateAccountSASQueryParameters(accountSASSignatureValues: Acc
 // @public
 export function generateQueueSASQueryParameters(queueSASSignatureValues: QueueSASSignatureValues, sharedKeyCredential: StorageSharedKeyCredential): SASQueryParameters;
 
-// @public
+// @public (undocumented)
 export interface GeoReplication {
     lastSyncOn: Date;
     status: GeoReplicationStatusType;
 }
 
 // @public
-export type GeoReplicationStatusType = 'live' | 'bootstrap' | 'unavailable';
+export type GeoReplicationStatusType = string;
```

Comment:
Are these marked as extensible enums in the SWAGGER?

Reply:
I wonder if they are running into the issue where if the swagger doesn't explicitly say modelAsString:false it would default to true.

If this is the case, maybe we can have a transform to add `modelAsString:false`?

Reply:
@joheredi I have checked. This is explicitly set to modelAsString to true. Unfortunately, the ```| string``` which is supposed to be added at the end by the V4 generator is missing due to the bug in it.

Reply:
Just to be clear, the intention here is to allow any string value?

Reply:
Yes

Reply:
I wonder whether this would cause breaking change as the type now is wider. Customer may have code that only handle the three cases?

Reply:
Per the service it is an extensible enum. So, we have to allow the expandable values.